### PR TITLE
Homework_1 first commit

### PR DESCRIPTION
**Pull request by ValeriiaPershakova**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -69,6 +98,18 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```
Comment (khda91) [inline pom.xml]: Why do you need `maven-install-plugin`?

**Review by khda91 — COMMENTED** [review]
please resolve comment
Project does not compile
Error:
```
[INFO] Scanning for projects...
[INFO] 
[INFO] --------------------< com.epam:ValeriiaPershakova >---------------------
[INFO] Building ValeriiaPershakova 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for com.epam:calculator:jar:1.0 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.237 s
[INFO] Finished at: 2019-11-23T15:14:50+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project ValeriiaPershakova: Could not resolve dependencies for project com.epam:ValeriiaPershakova:jar:1.0-SNAPSHOT: Failure to find com.epam:calculator:jar:1.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

File: pom.xml (hunk 1 above)
Comment (ValeriiaPershakova) [reply pom.xml]: It was made to avoid using system scope for dependency. As I understand now, it doesn't work because we never make "Install" stage

**Review by khda91 — APPROVED** [review]
Approved. Could be merged to the master branch

**Pull request merged**